README.md

```diff
@@ -146,6 +146,7 @@ Table of Contents
 
   * https://bitnami.com/ - Deploy prepared apps on IaaS. Management of 1 AWS micro instance free
   * http://www.visualops.io/ - 3,600 instance hours per month free
+  * http://devopsboard.com/ - Manage software and hardware issues aggregated from monitoring tools on single Kanban board
```

Comment:
does this have a free tier or is it just a trial? looks like just a trial to me which wouldnt fit the goal of this list